engine/src/main/java/org/destinationsol/game/ship/SolShip.java

```diff
@@ -695,4 +699,11 @@ public int getFactionID() {
     public void changeDisposition(int id) {
         FactionInfo.setDisposition(id, -1);
     }
+
+    public static void setInvincible(){
+        isInvincible = true;
+    }
+    public static void setNonInvincible(){
```

Comment:
Merge the two methods, using a parameter (e.g. call either `setInvincible(true)` or `setInvincible(false)`

Reply:
Also, you can probably log.warn if it is being set to the same value it already is.

Reply:
And add space before brace